scripts/bootstrap/src/metadata.script.ts

```
@@ -0,0 +1,169 @@
+import fs from "fs";
```

ESLint

[error] 'fs' is defined but never used.

(@typescript-eslint/no-unused-vars)